config/config.json

```diff
@@ -0,0 +1,10 @@
+{ 
+  // path to database file
+  "database_path":"/tmp/iroha_ledger",
```

Comment:
Follow a consistent naming style for configuration keys (either `underscore_lower_case` or `camelCase`).